Resources/Textures/Objects/Misc/stamps.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432. New stamp- icons taken from tgstation at https://github.com/tgstation/tgstation/commit/fb1012102257b7b0a08d861fd2b8ba963c416e93. stamp-warden is a darker version of stamp-hos. stamp-lawyer is a darker version of stamp-cap. paper_stamp-greytide is a grey and shaded in version of stamp-hos. paper_stamp-psychologist is a recolored stamp-cmo.",
+    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432. New stamp- icons taken from tgstation at https://github.com/tgstation/tgstation/commit/fb1012102257b7b0a08d861fd2b8ba963c416e93. stamp-warden is a darker version of stamp-hos. stamp-lawyer is a darker version of stamp-cap. paper_stamp-greytide is a grey and shaded in version of stamp-hos. paper_stamp-psychologist is a recolored stamp-cmo. paper_stamp-wizard is a recolored stamp-rd.",
```

wow i just looked at this copyright section and i hate it. its not your fault but it attributes no author. holy shit.
```suggestion
    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432. New stamp- icons taken from tgstation at https://github.com/tgstation/tgstation/commit/fb1012102257b7b0a08d861fd2b8ba963c416e93. stamp-warden is a darker version of stamp-hos. stamp-lawyer is a darker version of stamp-cap. paper_stamp-greytide is a grey and shaded in version of stamp-hos. paper_stamp-psychologist is a recolored stamp-cmo. paper_stamp-wizard is a recolored stamp-rd by brassicaprime69 (Discord)",
```

---

yeah i was confused as well